core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -67,14 +67,12 @@ const showMarketDetails = async (web3, artifacts, emp) => {
     };
   } else {
     const position = await emp.positions(sponsorAddress);
-    console.log("You have:");
-    console.log(
-      "Position: " +
-        fromWei(collateral) +
-        " WETH backing " +
-        fromWei(position.tokensOutstanding.toString()) +
-        " synthetic tokens"
-    );
+
+    console.table({
```

Comment:
@ptare do you want each row in a table to be selectable? Or is this table just displayed for information

Reply:
This table is just for information. But there are other places (e.g., listMarkets) where we just show a space separate string. It *might* be better to organize that into a table as well (or might not be worth it haha).

Is it possible to do that with inquirer?

Reply:
I would just look at plugins, for example, I don't think these are exactly what you want but perhaps they could be configured to be so:

https://github.com/eduardoboucas/inquirer-table-prompt
https://github.com/adam-golab/inquirer-select-line

Reply:
ye, I agree with @nicholaspai . if you want something selectable from within a table a plugin to inquire will be the way to go

Reply:
I think it's not worth it for now, if it's not just out of the box. We could always revisit later.